debian/wireguard-dkms.postinst

```diff
@@ -25,8 +25,8 @@ case "$1" in
 			echo "new version, you will need to remove the old module and load the new one. As"
```

Comment:
`echo "new version, you will need to remove the old module and load the new one."`
`echo "You can accomplish this with the following commands:"`